dScripts/ChooseYourDestinationNsToNt.cpp

```diff
@@ -59,14 +59,11 @@ void ChooseYourDestinationNsToNt::BaseChoiceBoxRespond(Entity* self, Entity* sen
 
         std::u16string strText = u"";
 
-        if (newMap == 1200)
-        {
+        if (newMap == 1200) {
             strText = u"UI_TRAVEL_TO_NS";
-        }
-        else
-        {
+        } else if (newMap == 1900) {
             strText = u"UI_TRAVEL_TO_NEXUS_TOWER";
-        }
+        } else strText = u"ZoneTable_" + GeneralUtils::ASCIIToUTF16(std::to_string(newMap)) + u"_summary";
```

Comment:
There is a GeneralUtils::to_u16string() if I recall correctly.

Reply:
Quite literally a few lines below that to_u16string is used :upside_down_face: silly me
Thanks for pointing it out!